jsonnet/kube-prometheus/components/k8s-control-plane.libsonnet

```diff
@@ -233,6 +233,35 @@ function(params) {
       }],
     },
   },
+  
+  podMonitorKubeProxy: {
+    apiVersion: "monitoring.coreos.com/v1",
+    kind: "PodMonitor",
+    metadata: {
+      labels: {
+        "k8s-app": "kube-proxy"
+      },
+      name: "kube-proxy",
+      namespace: k8s._config.namespace,
+    },
+    spec: {
+      jobLabel: "k8s-app",
+      namespaceSelector: {
+        matchNames: [
+          "kube-system"
+        ]
+      },
+      selector: {
+        matchLabels: {
+          "k8s-app": "kube-proxy"
+        }
+      },
+      podMetricsEndpoints: [{
+          honorLabels: true,
+          targetPort: 10249
+      }]
+    }
+  },
```

Comment:
Seems jsonnet fmt is not happy about this. Could you run `make fmt` and commit changes? No need to regenerate assets though ;)

Reply:
done 😄